support/SubmitScripts/Frontera.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 #SBATCH -J MyJobName       # Job name
-#SBATCH -o MyJobName.o%j   # Name of stdout output file
-#SBATCH -e MyJobName.e%j   # Name of stderr error file
+#SBATCH -o spectre.stdout  # Name of stdout output file
+#SBATCH -e spectre.stderr   # Name of stderr error file
 #SBATCH -p small           # Queue (partition) name - for 3+ nodes use 'normal'
```

Comment:
Answering a question that came up during review but didn't make it into the comments: the -p option is required.